packages/cms/pages/settings/profile/profile.tsx

```diff
@@ -122,6 +236,12 @@ export const Profile: FunctionComponent<ProfileProps> = () => {
               </div>
             </EuiDescribedFormGroup>
           </EuiForm>
+
+          <EuiGlobalToastList
+            toasts={toasts}
+            dismissToast={removeToast}
```

Comment:
The dismiss toast function will also come from the toasts store.

Reply:
okay